src/server/worldserver/worldserver.conf.dist

```diff
@@ -1945,6 +1945,13 @@ ListenRange.Yell = 300
 
 WaypointMovementStopTimeForPlayer = 120
 
+#    NpcEvadeIfTargetIsUnreachable
+#        Description: Specifies the time (in seconds) that a creature whom target
+#                     is unreachable to end up in evade mode.
+#        Default:     5
```

Comment:
Comparing this 2 videos:

AC: https://www.youtube.com/watch?v=fHGqFGWQ7q0

Retail (Classic): https://www.youtube.com/watch?v=i-tCzgBx1UQ

The default value shouldn't be 0?

Reply:
It's not ready yet, those are all 'thrown' in just to test. Will cleanup and prepare it in a couple of hours once I finish it.